paths/v2/results_bulk.yaml

```diff
@@ -1,4 +1,9 @@
 post:
+  summary: (Beta) Bulk create test run result.
+  description: |
+    This method allows to create a lot of test run result at once.
```

Comment:
```suggestion
    This method allows to create multiple test run results at once.
```

Reply:
It was copied from the bulk result endpoint's description